docs/repo/property-pages/README.md

```diff
@@ -1,6 +1,6 @@
 ﻿# Project Property Pages
 
-This documentation details the updated Project Properties UI and associated back end, added to Visual Studio in 2021.
+This documentation details the updated Project Properties UI and associated back end, added to Visual Studio in 2022.
```

Comment:
Talking about the product version is probably more useful than the year but we want to be clear on which one we mean.

```suggestion
This documentation details the updated Project Properties UI and associated back end, first released in Visual Studio 2022.
```